### PR TITLE
Functional test - Add test quick edit languages

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Add test 'Quick edit languages'
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | `TEST_PATH='functional/BO/11_international/01_localization/languages/01_filterAndQuickEditLanguages.js' URL_FO=shopUrl/ npm run specific-test`

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/19066)

### CODE REVIEW

File: tests/puppeteer/pages/BO/international/languages/index.js [hunk 1]
```diff
@@ -155,6 +158,32 @@ module.exports = class Languages extends LocalizationBasePage {
...
+    return this.elementVisible(this.enabledColumnValidIcon(row), 100);
+  }
+
+  /**
+   * Enable/Disable language
+   * @param row
+   * @param valueWanted
+   * @return {Promise<bool>}, true if click has been performed
+   */
+  async quiEditLanguage(row, valueWanted = true) {
```
Comment (SimonGrn) [inline tests/puppeteer/pages/BO/international/languages/index.js]:
```suggestion
  async quickEditLanguage(row, valueWanted = true) {
```

File: tests/puppeteer/campaigns/functional/BO/11_international/01_localization/languages/01_filterAndQuickEditLanguages.js [hunk 2]
```diff
@@ -169,4 +173,64 @@ describe('Filter Languages', async () => {
...
+    });
+
+    const tests = [
+      {args: {action: 'disable', enabledValue: false}},
+      {args: {action: 'enable', enabledValue: true}},
+    ];
+    tests.forEach((test) => {
+      it(`should ${test.args.action} first language`, async function () {
+        await testContext.addContextItem(this, 'testIdentifier', `${test.args.action}Language`, baseContext);
+        const isActionPerformed = await this.pageObjects.languagesPage.quiEditLanguage(1, test.args.enabledValue);
```
Comment (SimonGrn) [inline tests/puppeteer/campaigns/functional/BO/11_international/01_localization/languages/01_filterAndQuickEditLanguages.js]:
```suggestion
        const isActionPerformed = await this.pageObjects.languagesPage.quickEditLanguage(1, test.args.enabledValue);
```

File: tests/puppeteer/pages/BO/international/languages/index.js (hunk 1 above)
Comment (boubkerbribri) [reply tests/puppeteer/pages/BO/international/languages/index.js]: thanks

File: tests/puppeteer/campaigns/functional/BO/11_international/01_localization/languages/01_filterAndQuickEditLanguages.js (hunk 2 above)
Comment (SimonGrn) [reply tests/puppeteer/campaigns/functional/BO/11_international/01_localization/languages/01_filterAndQuickEditLanguages.js]: Not sure this is the only place you need to fix.

Comment (boubkerbribri) [reply tests/puppeteer/campaigns/functional/BO/11_international/01_localization/languages/01_filterAndQuickEditLanguages.js]: I fixed it in 2 places :+1:

**Review by PierreRambaud — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Pull request merged**